contributing.md

```diff
@@ -0,0 +1,80 @@
+issue.
+
+<!---For other issues, please email the Task 43 distribution list at `xx`.--->
+
+## Repository
+
+The digital WRA digital_wra_data_standard repository is hosted on Github, and located here: https://github.com/IEA-Task-43/digital_wra_data_standard
+
+This repository is organized using a modified git-flow system. Branches are organized as follows:
+
+- master: Stable release version. Must have good test coverage and may not have all the newest features.
+- develop: Development branch which contains the newest features. Tests must pass, but code may be unstable.
+- feature/xxx: Branch from develop, should reference a github issue number.
+
+To work on a feature, please fork digital_wra_data_standard first and then create a feature branch in your own fork.
+Work out of this feature branch before submitting a pull request.
+Be sure to periodically synchronize the upstream develop branch into your feature branch to avoid conflicts in the pull request.
+
+When the feature branch is ready, make a pull request to IEA-Task-43/digital_wra_data_standard through the Github.com UI.
+
+## Pull Request
+
+Pull requests must be made for any changes to be merged into release branches.
+They must include updated documentation and pass all unit tests and integration tests.
+In addition, code coverage should not be negatively affected by the pull request.
+
+**Scope:** Encapsulate the changes of ideally one, or potentially a couple, issues.
+It is greatly preferable to submit three small pull requests than it is to submit one large pull request.
+Write a complete description of these changes in the pull request body.
+
+**Tests:** TBD
+<!---Must pass all tests. Pull requests will be rejected if tests do not pass.
+Tests are automatically run through Github Actions for any pull request or push to the master or develop branches.-->
+
+**Documentation:** TBD
+<!---Include any relevant changes to inline documentation, as well as any changes to the RST files located in /sphinx.--->
+
+**Coverage:** TBD <!--- The testing framework (described below) will generate a coverage report. Please ensure that your
+work is fully covered. -->
+
+**Changelog:** For pull requests that encapsulate a user-facing feature, or is significant to users of digital_wra_data_standard for some other reason, please add a line to CHANGELOG.md in the [Unreleased] section.
+
+## Coding Style
+
+This code follows the PEP 8 style guide <!---and uses the ``pycodestyle`` linter to check for compliance.
+The only exception is the line length limit of 120 characters.
+
+```
+pylint --max-line-length=120 operational_analysis
+```
+-->
+## Documentation Style
+TBD
+D<!--- ocumentation is written using RST, and is located both inline and within the /sphinx directory.
+Any changes to the analysis methodology should be discussed there or offline. Once a methodology change is decided,
+create new tickets in this repository towards implementing the change.-->
+
+## Testing
```

Comment:
This is not a Python package, so is there really any code to test? If anything we can define guidelines for validating the schema.

Reply:
yawp.  this example was python centric good catch

Reply:
I have used a few validators to validate the schema. This was the main oneI used:
https://www.jsonschemavalidator.net/s/UMQf3zXk
I used the demo json file in https://github.com/IEA-Task-43/digital_wra_data_standard/blob/master/demo_data/iea43_wra_data_model.json to help validate it too. I literally just copy and pasted the schema and demo file in to see if they matched.

Another one was:
https://www.liquid-technologies.com/online-json-schema-validator

Maybe something around these could be sufficient for now?

Reply:
I agree. Why don't we say something along the lines of

> For any changes to the JSON schema, the contributor should
> - Make sure that the changes result in a valid JSON schema
> - Adjust the [demo file](https://github.com/IEA-Task-43/digital_wra_data_standard/blob/master/demo_data/iea43_wra_data_model.json) to reflect the changes
>
> Tools like [jsonschemavalidator](https://www.jsonschemavalidator.net) can help with these tasks.

Reply:
@flrs-edf  your edits seem reasonable.  whta do you guys think of this github action to help automate the process? https://github.com/marketplace/actions/validate-json 

It looks a little young as a project but might be valuable.  Otherwise happy to integrate the language you recommend above.

Reply:
I think go with Florian's text for now as this project would need to be validated first. I have created a new issue #11 to cover this.

Reply:
@Dynorat I agree with @stephenholleran's assessment. I have seen the action, but I don't feel confident about integrating it into this project since it does not seem very mature (yet).